test/test-PerfMeter.js

```diff
@@ -0,0 +1,162 @@
+/* globals Services */
+
+"use strict";
+const simplePrefs = require("sdk/simple-prefs");
+
+const {before, after} = require("sdk/test/utils");
+const tabs = require("sdk/tabs");
+const {setTimeout} = require("sdk/timers");
+const {ActivityStreams} = require("lib/ActivityStreams");
+const {Cu} = require("chrome");
+Cu.import("resource://gre/modules/Services.jsm");
+
+let ACTIVITY_STREAMS_URL;
```

Comment:
The code could be made simpler and easier to read in the tests by just passing a parameter instead of using a global state.